test/factories/internship_offers.rb

```diff
@@ -8,6 +8,7 @@
     max_candidates { 1 }
     blocked_weeks_count { 0 }
     sector { create(:sector) }
+    school_type { InternshipOffer.school_types.keys.sample.to_sym }
```

Comment:
```suggestion
    school_type { InternshipOffer.school_types.keys.first.to_sym }
```
eviter le random sur les tests, si derrieredes comportements changent on va galerer a comprendre prk certains tests cassent...